reference/tidb-binlog/monitor.md

```diff
@@ -2,7 +2,7 @@
 title: TiDB Binlog Monitoring
 summary: Learn how to monitor the cluster version of TiDB Binlog.
 category: reference
-aliases: ['/docs/tools/binlog/monitor/','/docs/v3.0/how-to/monitor/tidb-binlog/']
+aliases: ['/docs/tools/binlog/monitor/','/docs/v3.0/how-to/monitor/tidb-binlog/','/docs/dev/how-to/monitor/tidb-binlog/']
```

Comment:
As tested, this seems to the only available alias.
```suggestion
aliases: ['/docs/tools/binlog/monitor/']
```

Reply:
Many thanks!